pkg/config/config.go

```diff
@@ -50,6 +50,9 @@ func (c *Config) Init() {
 func (c *Config) Check() error {
 	var err error
 
+	// ensure that there is a value set for the --config flag, or that enough command line values
```

Comment:
I can't see any code added here, intentional?

Reply:
There is a full function here :-)

Reply:
I see nothing under the comment, just the previous code already there, intentional?

Reply:
I see what you mean now - have removed comment